### PR TITLE
Allow non-module reducers.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
This allows reducers outside of modules to be used with `ModuleProvider`.

### CODE REVIEW

**Comment by mboperator** [comment]
Awesome 👍 This provides the best of both worlds.

**Comment by mboperator** [comment]
Should this run the register reducers function once on `ModuleProvider` mount? So that the outside reducers are registered out the gate?

**Comment by wtgtybhertgeghgtwtg** [comment]
Not strictly doable on mount, since `registeredModules` cannot be accessed outside of `registerModule`, but it now does so before `registerModule` returns.

File: src/ModuleProvider/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  };
+
+  constructor(props, context) {
+    super(props, context);
+    // <Provider> does not support changing "store", so "store" should always be the same instance
+    this.registerModule = registerModule(
+      props.store,
+      props.combineReducers || combineReducers,
+      props.staticReducers);
+  }
```
Comment (mboperator) [inline src/ModuleProvider/index.js]: Can you instead call registerModule here to producing side effects when generating the `registerModule` function?

```js
this.registerModule(props.staticReducers)
```

**Comment by mboperator** [comment]
+1

**Pull request merged**